main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/weaveworks/release-note-action/action"
+)
+
+func main() {
+	a, err := action.New(releaseNotAction)
+	if err != nil {
+		fmt.Fprintln(os.Stdout, "failed to create action: %w", err)
```

Comment:
`fmt.Println` defaults to `os.Stdout` https://golang.org/src/fmt/print.go?s=7957:8006#L263

Also, you're not formatting the code, so the `%w` won't do anything, but even then, you don't want to wrap the error here?

I think you can simplify this to:

```go
fmt.Printf("failed to create action: %s\n", err)
```

Reply:
I agree. I was thinking of the tests in the future and attaching a different writer. But for the time being i have removed